java/code/webapp/WEB-INF/pages/yourrhn.jsp

```diff
@@ -4,33 +4,32 @@
 <%@ taglib uri="http://rhn.redhat.com/rhn" prefix="rhn" %>
 
 <html>
-<head>
+<body>
```

Comment:
Question: do you think this will impact the loading performance somehow? As a *forecast* by this practice the static page should load faster, but then the `DWR` resuests will start a bit later. Am I missing something? Btw, it's just to be aware of what's gonna change or not, nothing to prevent this change in the end. :wink:

Reply:
I don't think that will impact the performance in a noticeable way. 

I think we are only starting the ajax call in the <body> in TasksRenderer.renderAsync. 

Anyway, if you access the yourhn.jsp from another page it will be faster now and we also want to get rid of dwr